packages/styled-components/package.json

```diff
@@ -10,10 +10,33 @@
     "./dist/styled-components.esm.js": "./dist/styled-components.browser.esm.js",
     "./dist/styled-components.cjs.js": "./dist/styled-components.browser.cjs.js"
   },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": {
+        "node": "./dist/styled-components.esm.js",
+        "default": "./dist/styled-components.browser.esm.js"
+      },
+      "require": {
+        "node": "./dist/styled-components.cjs.js",
+        "default": "./dist/styled-components.browser.cjs.js"
+      }
```

Comment:
Those two share the same directory, they are meant to represent two different module formats (CJS vs ESM) and yet they are using the same extension (`.js`). This isn't correct and will cause problems one way or another. At the very least you need to either put those in different directories with appropriate `package.json` scopes or use different extensions for those files (with the current content of the `package.json` file you should use `.mjs` for your ESM modules)